### PR TITLE
Fix for UT failures

### PR DESCRIPTION
**Pull request by swathinsankaran**
This PR cherry-picks the UT fixes from the master branch.

### CODE REVIEW

**Review by apalsule — APPROVED** [review]

File: tests/evhtests/l7_evh_ingressclass_test.go [hunk 1]
```diff
@@ -400,20 +439,19 @@ func TestEVHUpdateInfraSettingInIngressClass(t *testing.T) {
 	// update infrasetting ref in ingressclass, model changes
 	g := gomega.NewGomegaWithT(t)
 
-	ingClassName, ingressName, ns, settingName1, settingName2 := "avi-lb", "foo-with-class", "default", "my-infrasetting", "my-infrasetting2"
+	ingClassName, ingressName, ns, settingName1, settingName2 := "avi-lb", "foo-with-class", "default", "my-infrasetting1", "my-infrasetting2"
```
Comment (akshayhavile) [inline tests/evhtests/l7_evh_ingressclass_test.go:442]: We have changed infrasetting name here. Are we still facing timing related issue here ?

File: tests/ingresstests/ingressclass_test.go [hunk 2]
```diff
@@ -432,20 +479,19 @@ func TestUpdateInfraSettingInIngressClass(t *testing.T) {
 	// update infrasetting ref in ingressclass, model changes
 	g := gomega.NewGomegaWithT(t)
 
-	ingClassName, ingressName, ns, settingName1, settingName2 := "avi-lb", "foo-with-class", "default", "my-infrasetting", "my-infrasetting2"
+	ingClassName, ingressName, ns, settingName1, settingName2 := "avi-lb", "foo-with-class", "default", "my-infrasetting1", "my-infrasetting2"
```
Comment (akshayhavile) [inline tests/ingresstests/ingressclass_test.go:482]: Same comment as above.

File: tests/evhtests/l7_evh_ingressclass_test.go (hunk 1 above)
Comment (swathinsankaran) [reply tests/evhtests/l7_evh_ingressclass_test.go:442]: No failures with the infrasetting here. My thought was to follow the same naming convention as `my-infrasetting2`.

File: tests/ingresstests/ingressclass_test.go (hunk 2 above)
Comment (swathinsankaran) [reply tests/ingresstests/ingressclass_test.go:482]: No failures with the infrasetting here. My thought was to follow the same naming convention as my-infrasetting2.

**Pull request merged**